ext/templates/puppetdb-ssl-setup.erb

```diff
@@ -226,7 +226,7 @@ else
   agent_confdir=`puppet agent --configprint confdir`
   agent_vardir=`puppet agent --configprint vardir`
 
-  if [ -d "/etc/puppetlabs/puppetdb" ] ; then
+  if id -u pe-puppetdb > /dev/null 2>&1; then
```

Comment:
tested id -u on debian 7, ubuntu 14.04, centos 7, sles 11, and PE master-supported rhel derivatives. It also works on OS X, which getent does not IIRC.